SU-HKKU/cardinal_service_profile
# CSD-8 Homepage filter banner support

# READY FOR REVIEW

# Summary
- Patch drupal core for some issues with views exposed filters in layout builder
- modify the view exposed filters blocks to add a action url input so that the form will submit to another page.

# Need Review By (Date)
- 5/12

# Urgency
- medium

# Steps to Test
1. `composer require SU-HKKU/cardinal_service_profile:dev-csd-8`
1. `blt drupal:sync --site=cardinalservice`
1. on any page edit the layout and place the block "Exposed form: su_opportunities-filtered_all"
1. populate the action url field
1. save the layout and submit the form.
1. verify it sends you to the url you entered as the action field.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
